Core/include/Acts/Fitter/GainMatrixSmoother.hpp

```diff
@@ -48,15 +47,15 @@ class GainMatrixSmoother {
   ///
   /// @return The smoothed track parameters at the first measurement state
   template <typename source_link_t>
-  Result<parameters_t> operator()(const GeometryContext& gctx,
-                                  MultiTrajectory<source_link_t>& trajectory,
-                                  size_t entryIndex) const {
+  Result<void> operator()(const GeometryContext& gctx,
+                          MultiTrajectory<source_link_t>& trajectory,
+                          size_t entryIndex) const {
     ACTS_VERBOSE("Invoked GainMatrixSmoother on entry index: " << entryIndex);
     using namespace boost::adaptors;
 
     // using ParVector_t = typename parameters_t::ParVector_t;
-    using CovMatrix = typename parameters_t::CovMatrix_t;
-    using GainMatrix = CovMatrix;
+    using Matrix_t =
```

Comment:
Following the naming convention, shall we just use `Matrix`?

Reply:
Changed